bundles/AdminBundle/public/js/pimcore/events.js

```diff
@@ -241,3 +241,9 @@
  */
 pimcore.events.preMenuBuild = "pimcore.menu.preBuild";
 
+/**
+ * after settings tab of page is opened
+ * layout and document are passed as parameter
+ */
+pimcore.events.assignTargetGroupToPage = "pimcore.page.assignTargetGroup";
```

Comment:
```suggestion
pimcore.events.prepareDocumentPageSettings = "pimcore.documentPage.prepareSettings";
```
this should go with making event generic